### DISCUSSION TITLE
Cannot delete glb files from Spoke project page

### DISCUSSION THREAD

**Discussion by ghost** [mozilla/hubs#4281] [Spoke Questions] [answered]
A few weeks ago when I was getting started with Spoke I imported a few glb models to my Spoke project page. Unlike other Spoke projects, I cannot delete these glb models, or at least I don't know how. They do not have the "three vertical dots" that other projects have that I would normally press to delete a project.

**Comment by Softvision-GeluHaiduc** [comment, marked as answer]
Thank you for logging this, I created an issue [#1146](https://github.com/mozilla/Spoke/issues/1146) in the spoke repository.
Marking this as answered, unmark the answer if you need any more information. Thank you!
